docs/en/stock-management/accounting/index.md

```diff
@@ -0,0 +1,163 @@
+
+Account Label | Number | Debit | Credit
+-- | -- | -- | --
+Cash | 57 |   | 105
+Supplier | 40 | 105 |
+
+
+### Purchasing with Cash
+In this scenario, the organisation purchases $105 of stock directly by paying cash to a shop.  In this case, OHADA does
+not require going through a supplier account.  Instead, the value of the medicine is directly expended from the cash account.
+The transaction looks like this:
+
+Account  Label | Number | Debit | Credit
+-- | -- | -- | --
+Cash | 57 | | 105
+Purchases of Goods | 601 | 105  |
+
+The only remaining step is to receive the goods in stock.  This is identical to the transaction written when receiving goods
+purchased on credit.
+
+Account Label | Number | Debit | Credit
+-- | -- | -- | --
+Variation of Goods | 603 |   | 105
+Stock | 30 | 105 |
+
+### After the Purchase
+It's important to note the balance of accounts in the OHADA chart of accounts after the purchase operation is completed.  In the more complex case
+of purchasing on credit, the balance of each account will be as follows:
+
+Number | Label | Debit | Credit | Balance
+-- | -- | -- | -- | --
+30 | Stock | $105.00 |   | $105.00
+40 | Supplier | $105.00 | $105.00 | $0.00
+57 | Cash | $0.00 | $105.00 | ($105.00)
+601 | Purchases of Goods | $105.00 |   | $105.00
+603 | Variation of Goods |   | $105.00 | ($105.00)
+  |   |   | Total | $0.00
+
+Since the sum of balances is $0.00, the general ledger is balanced.  The profit and loss statement (Compte d'exploitation) will show a net zero expense
+to the institution:
+
+Number | Label | Debit | Credit | Balance
+-- | -- | -- | -- | --
+601 | Purchases of Goods | $105.00 |   | $105.00
+603 | Variation of Goods |   | $105.00 | ($105.00)
+  |   |   | Net | $0.00
+
+Intuitively, this makes sense - the purchase of medicines converts cash assets into physical assets.  Those assets could be converted back to cash without
+a loss to the insitution.
+
+
+### Consumption of Medicines
+The consumption of medications under OHADA concerns only two accounts: the stock account (30) and the variation of goods account (603).  To exit stock $25.00 of stock,
+the stock account is credited and the variation account is debited.  The transaction is presented below.
+
+Account Label | Number | Debit | Credit
+-- | -- | -- | --
+Stock | 30 |  | 25
+Variation of Goods | 603 |  25 |
+
+## IFRS Stock Accounting
+
+In the IFRS accounting, purchasing medicines is never considered an expense, but the transfer of assets via an accounts payable.  Generally, a supplier issues an invoice
+for goods and the enterprise pays the supplier by cash or by bank.  The goods are entered into stock from the supplier's account.  So there are two transactions to consider:
+the invoicing for goods by the supplier and the payment for the goods by the enterprise.  Finally, when stock is consumed/destroyed/removed, the value of the stock is written
+to a cost of goods sold (COGS) expense account.
```

Comment:
Does BHIMA support IFRS stock accounting?

Reply:
Yes, possibly better that OHADA stock accounting.